test/visual-diff/visual-tests

```diff
@@ -33,6 +33,7 @@
   "asset_globs": [
     "examples/visual-tests/**",
     "examples/amphtml-ads/ads-tag-integration.js"
```

Comment:
missing , at the end

Reply:
hmm, shall we run lint on this file?

Reply:
I want to see it fails the job at the very least.

Reply:
yep. lint is a easier approach to catch this mistake
surfacing a proper error is going to be more helpful in long run.

I think we should do both